source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -46,6 +46,9 @@ void Filter::onComplete(const Status& status) {
   if (state_ == Responded) {
     return;
   }
+  if (context_) {
```

Comment:
Here, I don't think we should call context_->cancel().   It is called by verifier.  Context should be done.

Reply:
Not sure what you mean it is called by verifier. Context is  done here that's why I'm calling cancel.